nservicebus/azure-service-bus/transaction-support.md

```diff
@@ -7,37 +7,66 @@ tags:
+* Finally, it does not allow send operation on different messaging entities in the same transaction. So for example a send to Queue1 and a send to Queue2 in the same scope, will throw an exception. Again this problem is negated using separate instances of `IEnlistmentNotification` on each send operation. 
+
+The benefit of this approach is that user code can use database connections and send messages to multiple endpoints while maintaining the aspects expected from a transaction. However the downside is that each operation is individual. Even though they are executed at the same moment in a transaction, it can happen in theory that one of the operations fails, while the others do not. This problem is again mitigated by using multiple layers of retry behavior to reduce the chance, so a short interruption in connectivity or broker outage will not have a significant impact and operations will eventually succeed. However a full outage right after the database transaction committed could lead to a send operation that did not execute. This implies that business data could have been written, but outbound messages would not be sent out.
+
+NOTE: The maximum guarantee that the transport can deliver in these versions is in effect `ReceiveOnly`. 
+
+NOTE: Enabling/Disabling DTC will have no effect, this setting is ignored either way as Azure Service Bus does not support it.
+
+WARNING: Disabling transactions will also turn off the `PeekLock` mechanism, so that the transport immediately completes any incoming message before processing and shows no retry behavior. 
+  
+
+### Versions 7 and above
+
+In NServiceBus version 6 and above, the shape of the pipeline has changed so that a transport is in full charge of how a given transaction mode is implemented. A transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The Azure Service Bus transport only uses a transaction scope to implement the `SendAtomicWithReceive` transaction mode. Other modes work without any scope at all. The new architecture for `SendAtomicWithReceive` is schematically represented like this:
+
+[TODO: insert picture 2]
+
+Note the [fork in the pipeline](/nservicebus/pipeline/steps-stages-connectors.md) which is separating the user code invocation path from the dispatching path. By putting a suppress scope around this section of the pipeline, the transport can prevent any other transactional resource from enlisting in the scope.
+
+Furthermore the Azure Service Bus transport now also takes advantage of a little known capability if the Azure Service Bus SDK, the via entity path / transfer queue. Using this feature, send operations to different Azure Service Bus entities can be executed via a single entity, usually the receive queue. Schematically it works like this:
+
+[TODO: insert picture 3]
```

Comment:
The boundary of the Atomic operation is not clear. It looks like #1 is not involved. Perhaps to extend the box a little on the left?
![image](https://cloud.githubusercontent.com/assets/1309622/17801399/1efa24d2-65a7-11e6-999a-13cf7a7723ba.png)

Also, 4. Receive is not part of the atomic operation.  Why to enumerate and list it?